### ISSUE TITLE
Restrict follow operation to people located in France

### ISSUE THREAD

**Issue by Rayanox** [RayanoxPersonalProjects/Instagram-follower-booster#4]
The script "follow_user_followers" have to be executed on people from France to avoid going out of France and adding many foreign people.

To do so, there are several solutions:
 - Add random people from public french publications (Todo: add the REST link).
 - Detect the country of each last person followed and take the first one located in France (challenge = how to detect the person's country location ?)

**Comment by Rayanox** [comment]
Links are:
1) https://www.instagram.com/explore/locations/6889842/paris-france/?__a=1
2) https://www.instagram.com/explore/tags/paris/?__a=1

I prefer the first one, so I will use it.

**Comment by Rayanox** [comment]
For the REST client, use https://www.baeldung.com/guide-to-jayway-jsonpath

**Comment by Rayanox** [comment]
I found an alternate solution. It make me able to get recent people from France. It consists on getting datas from 'https://www.instagram.com/explore/tags/soiree/?__a=1', setting a french word as the tag ("soiree" here in the example). I must be sure that it's only a french word, which is the case for "soiree". So it works well, but with this method it's difficult to restrict to a smaller area.

If someone is interested to help and want to contribute with a better idea, PR will be welcome.
There are few other ways to get people from a specific location but it require to scrap Instagram. I decided to don't scrap because it is only for one feature and I don't have much time.
